topovis/topologytoolkit/src/datastructures/morsesmalecomplexdata.cpp

```diff
@@ -80,6 +80,58 @@ MorseSmaleComplexData::MorseSmaleComplexData(ttk::MorseSmaleComplex& msc,
                                 segmentation.msc.data());
 }
 
+CellType extremaDimToType(int dimensionality, char cellDim) {
+    if (dimensionality == 3) {
+        switch (cellDim) {
+            case 0:
+                return CellType::minima;
+            case 1:
+                return CellType::saddle;
+            case 2:
+                return CellType::saddle;
```

Comment:
It may be useful to differentiate between 1-saddles and 2-saddles. But for now, this is fine...